libflux/src/flux/scanner/mod.rs

```diff
@@ -76,6 +76,16 @@ impl Scanner {
         self._scan(2)
     }
 
+    // unread will reset the Scanner to go back to the Scanner's location
```

Comment:
I didn't make any changes to this method, I just moved it to be closer to the other public methods.